### DISCUSSION TITLE
Focus Indicators and Disabled Buttons and Img Alt

### DISCUSSION THREAD

**Discussion by kelsS** [RedHat-UX/red-hat-design-system#1708] [General]
Hi team! I reviewed the focus indicators for the newest versions of PFE and RHDS and I have some concerns about a few of them. I think these indicators need to be updated to be less subtle. I also have concerns about a specific blue over dark backgrounds that is really difficult and actually painful to look at. I am attaching my screenshots from RHDS only in this discussion. I will start another discussion in the PFE repo for PFE issues.

![Screenshot 2024-07-10 at 6 02 58 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/090376af-4b02-4eb9-99bd-f797adcfd7fb)

The dark blue in this image over that dark background is difficult to read and painful to look at.

![Screenshot 2024-07-10 at 6 00 44 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/195dd908-4bdc-4b3b-b142-e161c7195f45)

This seems like a bug with the demo of the table over dark backgrounds. Or we might be missing color styles for tables on dark backgrounds.

![Screenshot 2024-07-10 at 5 23 19 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/b5d28cd0-d5e4-42b0-ba21-83b2e8425e68)

For this one, I know that a tooltip pops up when the play button is in focus but it is really difficult to discern that dark blue against the black of the play button.

![Screenshot 2024-07-10 at 5 25 55 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/ba8ee88c-7424-461b-84c8-a97668ebc300)

This component needs to have an empty alt attribute when there is no alt text rather than no alt attr at all.

![Screenshot 2024-07-10 at 5 31 32 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/d10f797f-592b-4971-beba-6b9bb5e935d7)

![Screenshot 2024-07-10 at 5 54 16 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/0909d8da-7be6-426b-aff2-369e43f31ac4)

Disabled buttons should still be tab focusable. 

![Screenshot 2024-07-10 at 5 34 14 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/5ee68b8e-fcfb-455e-af12-cfeaf9e83b7d)

![Screenshot 2024-07-10 at 5 40 15 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/a32e35ec-9b7b-4a90-942a-56446010bdf3)

![Screenshot 2024-07-10 at 5 43 02 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/f289241a-f8d2-45e9-88e1-d248592660d8)

![Screenshot 2024-07-10 at 5 50 46 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/212aedb7-0e80-434c-a7c9-7660a1927d0a)

The light blue background for anywhere that a call to action is used is way too subtle and needs to be reworked.

![Screenshot 2024-07-10 at 5 35 00 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/1487e881-92c4-49fc-bce3-2c3fb8f2b73b)

![Screenshot 2024-07-10 at 5 35 26 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/a70b3579-87a9-4185-8694-babb2018e776)

![Screenshot 2024-07-10 at 5 37 34 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/92e1cdd7-355c-4267-b0a4-006f44618dee)

![Screenshot 2024-07-10 at 5 38 03 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/20908392/ead54ec2-d283-47a3-affa-4061376e2970)

The focus indicator for the brick variant is practically invisible and needs to be reworked.

**Comment by marionnegp** [comment]
@kelsS, thanks for bringing this up! We're releasing a [fix to the focus states for `<rh-cta>`](https://github.com/RedHat-UX/red-hat-design-system/issues/1528) variants in the upcoming release, which will also help with the issues in `<rh-card>`. Maybe auditing focus states for other elements and fixing them is something we can add to the next milestone, @markcaron @hellogreg?

**Comment by hellogreg** [comment]
Yeah, @kelsS and @marionnegp . There's some great stuff in here that may well spawn new issues! I think giving that stuff a look for the next milestone is a fine idea. I know some of those things (e.g., focusable disabled controls) were on Nikki's radar back in the day, too.
